common/src/main/scala/explore/model/reusability.scala

```diff
@@ -3,29 +3,32 @@
 
 package explore.model
 
+import java.time.Duration
+
+import clue.StreamingClientStatus
 import gem.Observation
+import gem.Observation
+import gem.util.Enumerated
 import gem.util.Enumerated
+import gpp.util.EnumZipper
 import gpp.util.Zipper
 import japgolly.scalajs.react.CatsReact._
 import japgolly.scalajs.react.Reusability
-import gem.util.Enumerated
-import gem.Observation
-import java.time.Duration
-import clue.StreamingClientStatus
 
 /**
   * Reusability instances for model classes
   */
 object reusability {
-  implicit val statusReuse: Reusability[StreamingClientStatus]     = Reusability.derive
-  implicit val durationReuse: Reusability[Duration]                = Reusability.by(_.getSeconds)
-  implicit val obsIdReuse: Reusability[Observation.Id]             = Reusability.by(_.format)
-  implicit val siderealTargetReuse: Reusability[SiderealTarget]    = Reusability.byEq
-  implicit val expTargetReuse: Reusability[ExploreSiderealTarget]  = Reusability.derive
-  implicit val expObsReuse: Reusability[ExploreObservation]        = Reusability.derive
-  implicit def enumReuse[A: Enumerated]: Reusability[A]            =
+  implicit val statusReuse: Reusability[StreamingClientStatus]             = Reusability.derive
+  implicit val durationReuse: Reusability[Duration]                        = Reusability.by(_.getSeconds)
+  implicit val obsIdReuse: Reusability[Observation.Id]                     = Reusability.by(_.format)
+  implicit val siderealTargetReuse: Reusability[SiderealTarget]            = Reusability.byEq
+  implicit val expTargetReuse: Reusability[ExploreSiderealTarget]          = Reusability.derive
+  implicit val expObsReuse: Reusability[ExploreObservation]                = Reusability.derive
+  implicit def enumReuse[A: Enumerated]: Reusability[A]                    =
```

Comment:
Do we have a summon method to just do `Enumerated[A]`?

Reply:
Good catch! Done.